manifests/cli_helper.pp

```diff
@@ -2,7 +2,16 @@
 #
 # A helper script for creating resources via the Jenkins cli
 #
-class jenkins::cli_helper {
+# Parameters:
+#
+# ssh_keyfile = undef
+#   Defaults to the value of $::jenkins::cli_ssh_keyfile. This parameter is
+#   deprecated, please set $::jenkins::cli_ssh_keyfile instead of setting this
+#   directly
+#
+class jenkins::cli_helper (
+  $ssh_keyfile = $::jenkins::cli_ssh_keyfile,
```

Comment:
I cannot remove that without breaking API compatibility, but this does remind me  that I was going to put an `info()` message about the parameter being deprecated